scripts/foundation-wallet-deployment/config.ts

```diff
@@ -0,0 +1,46 @@
+interface FoundationWalletDeploymentConfig {
+  beneficiary: string;
+  startTimestamp: number;
+  durationSeconds: number;
+  l2ArbitrumGovernor: string;
+  l2GovProxyAdmin: string;
+}
+
+interface ChainIDToConfig {
+  [key: number]: FoundationWalletDeploymentConfig;
+}
+
+const secondsInADay = 60 * 60 * 24;
+
+const chainIDToConfig: ChainIDToConfig = {
+  42161: {
+    beneficiary: "", // TODO
+    startTimestamp: 0, // TODO
+    durationSeconds: secondsInADay * (365 * 4 + 1), // Days in 4 years, includes 1 day for leap day
+    l2ArbitrumGovernor: "0xf07DeD9dC292157749B6Fd268E37DF6EA38395B9",
+    l2GovProxyAdmin: "0xdb216562328215E010F819B5aBe947bad4ca961e",
```

Comment:
I think we should load this from a single canonical reference (SDK or `files/mainnet/deployedContracts.json`)